diff --git a/debian/rules b/debian/rules
index 225c743c5..9e56de260 100755
--- a/debian/rules
+++ b/debian/rules
@@ -15,6 +15,7 @@ DEB_BUILD_GNU_TYPE=$(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
 
 CONFIG_OPTS = --prefix=/usr \
             --includedir=/usr/include/xenomai \
+            --sysconfdir=/etc \
             --mandir=/usr/share/man \
             --with-testdir=/usr/lib/xenomai/testsuite \
             --enable-lazy-setsched \
diff --git a/debian/xenomai-runtime.install b/debian/xenomai-runtime.install
index 77a4c445a..a9effa984 100644
--- a/debian/xenomai-runtime.install
+++ b/debian/xenomai-runtime.install
@@ -5,3 +5,5 @@ usr/bin/wf_generate
 usr/bin/wrap-link.sh
 usr/bin/xeno
 usr/sbin
+etc/rtnet.conf
+etc/tdma.conf
diff --git a/kernel/drivers/net/stack/include/rtnet_rtpc.h b/kernel/drivers/net/stack/include/rtnet_rtpc.h
index 01d7ab152..f4ad47306 100644
--- a/kernel/drivers/net/stack/include/rtnet_rtpc.h
+++ b/kernel/drivers/net/stack/include/rtnet_rtpc.h
@@ -43,7 +43,7 @@ struct rt_proc_call {
 	atomic_t ref_count;
 	wait_queue_head_t call_wq;
 	rtpc_cleanup_proc cleanup_handler;
-	char priv_data[0] __attribute__((aligned(8)));
+	char priv_data[] __attribute__((aligned(8)));
 };
 
 #define CALL_PENDING 1000 /* result value for blocked calls */
