diff --git a/debian/rules b/debian/rules
index 225c743c5..9e56de260 100755
--- a/debian/rules
+++ b/debian/rules
@@ -15,6 +15,7 @@ DEB_BUILD_GNU_TYPE=$(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
 
 CONFIG_OPTS = --prefix=/usr \
             --includedir=/usr/include/xenomai \
+            --sysconfdir=/etc \
             --mandir=/usr/share/man \
             --with-testdir=/usr/lib/xenomai/testsuite \
             --enable-lazy-setsched \
diff --git a/debian/xenomai-runtime.install b/debian/xenomai-runtime.install
index 77a4c445a..a9effa984 100644
--- a/debian/xenomai-runtime.install
+++ b/debian/xenomai-runtime.install
@@ -5,3 +5,5 @@ usr/bin/wf_generate
 usr/bin/wrap-link.sh
 usr/bin/xeno
 usr/sbin
+etc/rtnet.conf
+etc/tdma.conf
diff --git a/kernel/drivers/net/stack/rtnet_rtpc.c b/kernel/drivers/net/stack/rtnet_rtpc.c
index cd5f0546d..caed27269 100644
--- a/kernel/drivers/net/stack/rtnet_rtpc.c
+++ b/kernel/drivers/net/stack/rtnet_rtpc.c
@@ -90,7 +90,7 @@ int rtnet_rtpc_dispatch_call(rtpc_proc proc, unsigned int timeout,
 	if (call == NULL)
 		return -ENOMEM;
 
-	memcpy(call->priv_data, priv_data, priv_data_size);
+	unsafe_memcpy(call->priv_data, priv_data, priv_data_size, /*embedded variable length data*/);
 
 	call->processed = 0;
 	call->proc = proc;
